web/geo/generateZoneBoundingBoxes.ts

```diff
@@ -72,18 +86,44 @@ for (const zone of zonesGeo.features) {
   ];
 }
 
+if (isAggregate) {
+  const averageBoundingBox: number[][] = Object.values(boundingBoxes).reduce(
```

Comment:
Huh I thouth ESLint would complain about the usage of reduce.

Reply:
Replaced with a simple loop :)

Reply:
Do you want to do the same for the arrows? 😉
![image](https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/f9d803e7-b2eb-4005-98fb-106ac5529e8b)

Reply:
Eeeh, let's save that for another day 😅